src/App.js

```diff
@@ -41,9 +61,14 @@ class App extends Component {
   render() {
     return (
       <div className="app-wrapper">
-        <Route exact path="/" component={AsyncLanding} />
-        <Route exact path="/category/:category" component={AsyncLanding} />
-        <Route exact path="/category/:category/:id" component={AsyncItemView} />
+        <Suspense fallback={<div>Loading...</div>}>
+          <Route exact path="/" component={LazyLanding} />
+          <Route exact path="/category/:category" component={LazyLanding} />
+          <Route exact path="/category/:category/:id" component={LazyItemView} />
+          {/* ray test touch < */}
```

Comment:
I'm seeing quite a few `ray test touch` comments listed throughout the PR (I believe were left over from before). Would you mind if we removed these?

Reply:
Yes for sure, we must remove them because those are my dev-in-progress highlight comments.
Before I created the PRs I check them and remove. On my side I can't find those comments.
Let me double check remove them permanently.